math/src/traits.rs

```diff
@@ -24,6 +24,15 @@ pub trait ByteConversion {
         Self: std::marker::Sized;
 }
 
+/// Serialize function without options
+/// Used to serialize data to feed it into Fiat Shamir
+/// in some protocols, when ByteConversion is not avaible
+/// For example, when using Curve Points
+pub trait SimpleSerialization {
```

Comment:
Can we name this `DefaultSerialization` or maybe more specific `FiatShamirSerialization`? Or maybe just `Serialization`? Also I would use an adjective (Serializable) or verb for the trait name.

Reply:
It may be used somewhere else so I'd avoid `FiatShamirSerialization`. I wanted to avoid a plain `Serializable` since there are additional serialization functions, and I don't want to call the function `serialize()` to avoid collision of names. I like the default thought and the serializable by they seem hard to mix. 

We could call it`Serializable`, and the function `serialize`, and rename the current `serialize` to `serialize_with_args` or something like that.